docs/content/concepts/configuration/config-schema-legacy.mdx

```diff
@@ -50,8 +50,8 @@ def op_using_config(context: OpExecutionContext):
 
 @asset(config_schema={"person_name": str})
 def asset_using_config(context: AssetExecutionContext):
-    # Note how asset config is also accessed with context.op_config
-    return f'hello {context.op_config["person_name"]}'
+    # Note how asset config is accessed with context.op_execution_context.op_config
+    return f'hello {context.op_execution_context.op_config["person_name"]}'
```

Comment:
Unfortunately I think are stuck with the name `op_config` IMO because the underlying reality is that we only have config at op-level rather than at the asset-level. If you write a `multi_asset`, you can't have different asset keys with different config in that case. I think having a `config` property on `AssetExecutionContext` implies that you do.